src/backend/src/services/description-bullets.services.ts

```diff
@@ -0,0 +1,57 @@
+import { User, WBS_Element_Status } from '@prisma/client';
+import prisma from '../prisma/prisma';
+import { hasBulletCheckingPermissions } from '../utils/description-bullets.utils';
+import { AccessDeniedException, HttpException, NotFoundException } from '../utils/errors.utils';
+import { descBulletTransformer } from '../transformers/description-bullets.transformer';
+import { descBulletArgs } from '../prisma-query-args/description-bullets.query-args';
+
+export default class DBService {
+  static async checkDescriptionBullet(userId: number, descriptionId: number, user: User) {
```

Comment:
we don't want userId at all anymore (even in the route). The way we get the user (`getCurrentUser`) is through auth and the way we got userId was just the frontend sending it which is not secure. so we can get rid of userId and also make user the first argument for consistency with other services